src/coreclr/tools/Common/CommandLineHelpers.cs

```diff
@@ -96,6 +96,7 @@ public static TargetArchitecture GetTargetArchitecture(string token)
                     Architecture.Arm => TargetArchitecture.ARM,
                     Architecture.Arm64 => TargetArchitecture.ARM64,
                     Architecture.LoongArch64 => TargetArchitecture.LoongArch64,
+                    (Architecture)9 => TargetArchitecture.Riscv64,
```

Comment:
I think `Architecture.RiscV64` is supported now.

Reply:
We'll have to wait for new architecture to get to sdk and for this new sdk to be used in main branch build. Currently .net-8.0.1 sdk is used for main branch, so it'll take some time until .net9-preview1 sdk arrives (changes from #90203 are not present in .net8 branch). Maybe I'm wrong here, if so please correct me.

Reply:
My mistake. Thank you!